### DISCUSSION TITLE
Date / Timezone not the same as the server

### DISCUSSION THREAD

**Discussion by eglyn** [requarks/wiki#6857] [Help / Questions] [answered]
Hi all,
 
My server (Ubuntu 22.04 LTS) is synchronised with a NTP server.
I used this procedure to install WikiJS: https://docs.requarks.io/install/ubuntu

But, it seems WikiJS is 6 hours earlier than my server, and I don't find where to setup the time zone :/
There is no such option in the administrative console...

How can I sync Wiki with the server date ? 

Thx !

**Comment by NGPixel** [comment, marked as answer]
You can change your account timezone from the profile page.

**Reply by 2257396011** [reply]
Hello, I have found the way to set the time zone, but the displayed time is still incorrect. On my internet-enabled Windows machine, I set the time zone to Shanghai, and my local computer time is also set to Shanghai, but the time displayed is one hour earlier than my time. However, when I deployed on my offline Linux server, I also set both the server time zone and the time zone of wiki.js to Shanghai, but the time displayed was even earlier, and it did not match at all. Moreover, I conducted a test on these two machines. No matter how I modified the time, the displayed time after my editing remained the same, and it was not apparent what effect modifying the time zone had.
What is the relationship between the server time zone and the wiki time zone here? How can I set the time correctly?
Thank you!
